docs/tutorials/web/excel-addin-bert-js.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Custom Excel Functions for BERT Tasks in JavaScript
+description:  Custom Excel Functions for BERT Tasks in JavaScript
+parent: Deploy on web
+grand_parent: Tutorials
+has_children: false
+nav_order: 2
+---
+
+# ONNX Runtime Custom Excel Functions for BERT NLP Tasks in JavaScript
+{: .no_toc }
+
+In this tutorial we will look at how we can create custom excel functions (`ORT.Sentiment()` and `ORT.Question()`) to implement BERT NLP models with ONNX Runtime Web to enable deep learning in spreadsheet tasks. The inference happen locally in the browser with excel on the web. 
+
+<div class="embed-container">
+  <iframe
+      src="https://www.youtube.com/embed/wuSxWGY_Sjg"
+      width="560" height="315"
+      frameborder="0"
+      allowfullscreen="true">
+  </iframe>
+</div>
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Prerequisites
+
+- [See the prerequisites from this tutorial](https://learn.microsoft.com/en-us/office/dev/add-ins/tutorials/excel-tutorial-create-custom-functions?source=recommendations&tabs=excel-windows#prerequisites)
+- [ONNX Runtime Web](https://onnxruntime.ai/docs/get-started/with-javascript.html#:~:text=ONNX%20Runtime%20Web%20You%20can%20install%20and%20import,syntax%20%28recommended%29%20import%20%2A%20as%20ort%20from%20%27onnxruntime-web%27%3B)
+- This plugin currently doesn't work in the client version of excel.
+
+## What are Custom Functions?
+
+Excel has many native functions like `SUM()` that you are likely familiar with. Custom functions are a useful tool to create and add new functions to Excel by defining those functions in JavaScript as part of an add-in. These functions can be accessed within Excel just as you would any native function in Excel.
+
+## Creating the Custom Function project with the Yeoman CLI
+
+Now that we know what custom functions are lets look at how we can create functions that will inference a model locally to get the sentiment text in a cell or extract information from a cell by asking a question and the answer being returned to the cell.
+
+- The first thing we need to do is create the template project from the Yeoman cli. Follow the steps [here to create the base project](https://learn.microsoft.com/en-us/office/dev/add-ins/tutorials/excel-tutorial-create-custom-functions?source=recommendations&tabs=excel-online#create-a-custom-functions-project).
+
+- Install ONNX Runtime `npm install onnxruntime-web --save`
+
+- Once you have followed the steps to create the base project and tested that it works, then its time to start updating it for our custom function logic. You can run the project with the below command. This will start Excel web and side load the add-in to the spreadsheet that is provided in the command.
+
+```bash
+// Command to run on the web.
+npm run start:web -- --document {url}
```

Comment:
Also might want to highlight that Developer mode needs to be enabled

![image](https://user-images.githubusercontent.com/3302433/205407779-d8ad9524-2f76-49a5-bc74-e5e0bea6d223.png)

Reply:
I got this when I tried to run it

![image](https://user-images.githubusercontent.com/3302433/205408309-93254583-96b2-4897-9c26-0a364ece0968.png)

But I think that was because I had to manually accept the certificate? Then I had to kill the job and start it again.

Once it was successfully loaded I couldn't see where I could find the example custom functions?

Reply:
Ok got it finally. Found the namespace. I don't think it would hurt to spell these things out for a naïve user, so that they can test the basic concepts before digging into the ORT stuff

Reply:
These issues are covered in the tutorial that is linked in the prerequisite. I do not think it needs to be duplicated in this tutorial. This is focused on the ORT and inference logic on top of the tutorial for making a plugin.

Reply:
I think this tutorial would be better as a self contained article. Taking the user out of this context and into the middle of the other tutorial blurs the clarity of this one (even more so, as the steps are pretty simple, and not too much work to bring the steps into this one). We could have the other tutorial as a reference if folks want more background

Reply:
Updated!